tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	parser "github.com/kubevirt/monitoring/pkg/metrics/parser"
+	"kubevirt.io/ssp-operator/internal/operands/metrics"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+var excludedMetrics = map[string]struct{}{
+	"kubevirt_ssp_operator_up_total":           {},
+	"kubevirt_ssp_template_validator_up_total": {},
+}
+
+func readMetrics() []*dto.MetricFamily {
```

Comment:
why readMetrics() and not ReadMetrics()?

Reply:
Because it's not exported to other packages, no need to

Reply:
Sure I used big case since we use this function in the json generator file.
But it's OK with me to leave as is

Reply:
In Go you should use upper camel case only if you export it to other packages, not files. In this case, `readMetrics()` is from the same package.